doc/lettable-operators.md

```diff
@@ -95,11 +95,13 @@ const takeEveryNth = (n: number) => <T>(source: Observable<T>) =>
     })
   });
 
+const takeEveryNthSimple = (n: number) => <T>(source: Observable<T>) =>
```

Comment:
Interestingly, you could simplify this further like so:

```ts
const takeEveryNthSimplest = (n: number) => filter((_, index) => index % n === 0);
```

... the primary point here was to show the deepest example in the simplest way... But I suppose we can go into more detail.

Reply:
LGTM